extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/StorkConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.stork;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigDocSection;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * Stork configuration root.
+ */
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class StorkConfiguration {
+
+    /**
+     * ServiceDiscovery configuration for the service
+     */
+    @ConfigItem
+    @ConfigDocSection
+    @ConfigDocMapKey("service-name")
+    public Map<String, StorkServiceDiscoveryConfiguration> serviceDiscovery;
```

Comment:
I think it would be more clear if there was a single map here serviceName -> StorkServiceConfiguration
And StorkServiceConfiguration would have load balancer confdiguration and service discovery configuration